src/Notifier.h

```diff
@@ -8,17 +8,19 @@
 #include <mutex>
 #include <memory>
 #include <vector>
-
+#include <atomic>
 class Shotgun;
 class Vampire;
 
 class Notifier {
  public:
-   static std::unique_ptr<Notifier>  CreateNotifier(const std::string& notifierQueue, const std::string& handshakeQueue, const size_t handshakeCount);
+   static std::unique_ptr<Notifier>  CreateNotifier(const std::string& notifierQueue, const std::string& handshakeQueue, const size_t handshakeCount, size_t maxTimeoutInSec = 60, unsigned int maxGetShotTimeoutInMs = 10);
    size_t Notify(const std::vector<std::string>& messages);
    size_t Notify(const std::string& message);
    size_t Notify();
    virtual ~Notifier();
+   std::string ReceiveData();
```

Comment:
Please add unit tests to this repository for the new functions and any changed behavior